### DISCUSSION TITLE
Great Work!

### DISCUSSION THREAD

**Discussion by tcourchaine99** [xmiguel911x/Surface-Pro-4-i7-i3-Opencore#3] [General]
Great job! Using your config as a base I was able to get my SP4 running with the obvious exception of wifi & bluetooth. I changed a couple things for my config (i.e. using platform ID 02002659 instead of 00001659) but for the most part it's the same as yours.

FYI for yourself or anyone else who encounters this...in my case withthis device, for some reason Youtube does not like this IGPU (regardless of which platform ID you use) and it freezes your device to the point that you have to perform a forced shutdown. The only way I found to get around this was to turn off hardware accelartion in the browser (since boot arg -disablegfxfirmware didn't work). Safari is a lost cause as it's enabled by default on the recent versions with no way to turn it off but Chrome, Edge, Brave and other browsers that allow you the option to turn it off all work for this issue.

**Comment by xmiguel911x** [comment]
thank you very much for your solution
